velox/functions/prestosql/tests/MapTest.cpp

```diff
@@ -99,6 +99,26 @@ TEST_F(MapTest, noNulls) {
   assertEqualVectors(expectedMap, result);
 }
 
+TEST_F(MapTest, emptyMap) {
+  auto size = 1'000;
+  // We use unknown types here because no way to specify the output type for the
+  // map() call.
+  const TypePtr& emptyMapType = MAP(UNKNOWN(), UNKNOWN());
+  SelectivityVector rows(size);
```

Comment:
This variable is not needed. The only usage is rows.end() which can be replaced with size.